dist/templates/ovnkube-db.yaml.j2

```diff
@@ -48,6 +48,7 @@ spec:
   template:
     metadata:
       labels:
+        db-pod: "true"
```

Comment:
I think we should use an ovn specific name to avoid any label collisions down the road with users. Maybe "ovn-db-pod" would work?